src/app/pages/property-details/property-details.component.html

```diff
@@ -1,10 +1,35 @@
 <body>
+    <div class="confirm_delete" *ngIf="displayedModal">
+        <app-confirmation-modal (closeModalOutput)="closeModal($event)"></app-confirmation-modal>
+    </div>
     <div class="container">
         <div class="row">
             <div class="col-md-8">
                 <app-property-detail [title]="title" [street]="street" [state]="state" [city]="city" [price]="price"
                     [imageMain]="imageMain" [imageOthers]="imageOthers" [bathrooms]="bathrooms" [garages]="garages"
                     [bedrooms]="bedrooms" [ifBuilding]="ifBuilding"></app-property-detail>
+                <div class="col-md-12">
+                    <div class="row buttonsDiv" *ngIf="isAuthenticated">
+                        <span class="col">
+                            <button type="submit" class="btn btn-primary">
+                                <i class="fa fa-star"></i>
+                                Save
+                            </button>
+                        </span>
+                        <span class="col">
+                            <button type="submit" class="btn btn-primary">
+                                <i class="fa fa-edit"></i>
+                                Edit
+                            </button>
+                        </span>
+                        <span class="col">
```

Comment:
Awesome work on this feature @MBenedicte. It works as expected, however, It would be better to have only the client admin be the one that can see the `delete` option. This aims at having better UX for our product. 
There's a package we can make use of to achieve this. [ngx-permissions
](https://www.npmjs.com/package/ngx-permissions)

Reply:
Hey @Rhytah , How would this package work with the current implementation keeping in mind that a user token only returns the email, id and the expiry date, do we have a way of knowing the role of the currently logged in user when they view the details of any property?

![image](https://user-images.githubusercontent.com/12128153/64506329-e4d1b780-d2df-11e9-8d0a-59e4e58836bd.png)

Reply:
@Rhytah We don't have a way to know a user who is logged in. But a client cannot delete the property that us not his

Reply:
@MBenedicte I think we will sync up about this.

Reply:
Worked on this

Reply:
@MBenedicte Thanks for implementing this the best way possible